instrumentation/lettuce-4.3/src/main/java/com/lambdaworks/redis/AbstractRedisAsyncCommands.java

```diff
@@ -0,0 +1,55 @@
+package com.lambdaworks.redis;
+
+import com.lambdaworks.redis.api.StatefulConnection;
+import com.lambdaworks.redis.protocol.AsyncCommand;
+import com.lambdaworks.redis.protocol.ProtocolKeyword;
+import com.lambdaworks.redis.protocol.RedisCommand;
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Segment;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.fit.lettuce.instrumentation.NRBiConsumer;
+
+@Weave(type = MatchType.BaseClass)
+public abstract class AbstractRedisAsyncCommands<K, V> {
```

Comment:
Same comment about naming weaved library classed with `_Instrumentation` and specifying the `originalName` in the `Weave` annotation.